docs/1___overview.adoc

```diff
@@ -170,7 +170,7 @@ The following changes were performed as part of the 2.0.0 major release:
      This feature enables more advanced graphical editing of SSP models.
 * Traceability and meta-data features
   ** Integration of the `MetaData` element developed as part of the SSP Traceability layered standard into the core standard.
-     This allows the integration of arbitrary meta-data and digital signatures into the SSP file formats.
+     This allows the integration of arbitrary meta-data and digital signatures, such as life-cycle information, into the SSP file formats.
```

Comment:
Given that for life-cycle information we are more looking towards SSP Traceability, and meta-data really can be any arbitrary meta-data at all, I'd avoid pointing exactly to life-cycle information here.